lib/hca/enrollment_system.rb

```diff
@@ -865,18 +865,18 @@ def add_attachment(file, id, is_dd214)
     end
 
     # @param [Hash] veteran data in JSON format
-    # @param [Account] current_user
+    # @param [Hash] user_identifier
     # @param [String] form_id
```

Comment:
`current_user` was not a record, but instead a Hash. I changed the name of this variable because it was kind of misleading.